lib/langchain/llm/aws_bedrock/antrhopic_messages.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module Langchain::LLM
+  # LLM interface for Aws Bedrock Claude Messaging APIs: https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-anthropic-claude-messages.html
+  #
+  # Gem requirements:
+  #    gem 'aws-sdk-bedrockruntime', '~> 1.1'
+  #
+  # Usage:
+  #    claude3 = Langchain::LLM::AwsBedrock::AnthropicMessages.new(llm_options: {})
+  #
+  class AwsBedrock::AnthropicMessages < AwsBedrock
```

Comment:
Is it possible to re-use the existing [AwsBedrock](https://github.com/andreibondarev/langchainrb/blob/main/lib/langchain/llm/aws_bedrock.rb) class?

Reply:
@mtrefilek I think combining it under the same [AwsBedrock](https://github.com/andreibondarev/langchainrb/blob/main/lib/langchain/llm/aws_bedrock.rb) class makes sense.

You'd basically add:
```ruby
  class AwsBedrock < Base
    DEFAULTS = {
      completion_model_name: "[anthropic.claude-v2](anthropic.claude-3-sonnet-20240229-v1:0)",
```

And then the chat method only works with that model from my understanding.

Reply:
@andreibondarev Good call, It actually looks like all Claude Models support this new messaging inference request. I updated the default model and forced chat to only allow messages. 

I will add a condition to error .complete on claude-3 models

To my understanding no. I believe the models should all be better and cheaper based on most reading I have seen.